Multihost/basic-attestation/test.sh

```diff
@@ -207,7 +217,7 @@ Agent() {
         id keylime && rlRun "chown -R keylime.keylime ${SECUREDIR}"
 
         # common configuration goes here
-        rlRun "limeUpdateConf general receive_revocation_ip ${VERIFIER_IP}"
+        rlRun "limeUpdateConf agent revocation_notification_ip ${VERIFIER_IP}"
```

Comment:
Could you please move this line to the agent configuration section and also remove `# common configuration goes here`

Reply:
Sure, I'll fix this.